src/styles/App.css

```diff
@@ -0,0 +1,253 @@
+@import url("https://fonts.googleapis.com/css2?family=Montserrat:wght@100;400;700&family=Roboto+Slab:wght@400;700&display=swap");
+
+:root {
+  --blue: #1e90ff;
+  --white: #fff;
+  --blue1: #307bbe;
+  --blue2: #379cf6;
+  --black: #333;
+  --cream: #e8e8e8;
+}
+
+*,
+*::before,
+*::after {
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+}
+
+html {
+  font-size: 62.5%;
+  scroll-behavior: smooth;
+}
+
+body {
+  width: 100vw;
+  overflow-x: hidden;
+  height: 1097px;
+  margin: 0 40px 65px 0;
+  padding: 0 0 794px;
+  border-radius: 4px;
+  box-shadow: 0 0 60px 0 var(--black);
+  background-color: var(--white);
+  font-weight: 400;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+}
+
+.App {
+  width: 90%;
+}
+
+.navbar {
+  width: 100%;
+  display: flex;
+  justify-content: space-around;
+  align-items: center;
```

Comment:
- [ ] Align-items :center has also been repeated. Let's keep it DRY by applying the concepts of display:flex i demonstrated above